Code-Hammers/code-hammers
# [CHE-140] Create Job Application Tracker

### Description

This PR merges the Job Application Tracker foundational code into the dev branch.

### Jira Task

[CHE-140](https://codehammers.atlassian.net/browse/CHE-140?atlOrigin=eyJpIjoiZTg1YzRlZDUzYjZhNDdiMGJlNDdmZTM4ZThjZTFjMzAiLCJwIjoiaiJ9)

### Testing Instructions

1. Ensure branches are up to date
2. Spin up dev environment and use the application tracker. 

### Checklist

#### All Team Members


### Additional Notes, Images, etc.

N/A


[CHE-140]: https://codehammers.atlassian.net/browse/CHE-140?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
